CMakeLists.txt

```diff
@@ -1,6 +1,9 @@
 cmake_minimum_required(VERSION 3.16.3)
+if(COMMAND CMAKE_POLICY)
+  cmake_policy(SET CMP0135 NEW)
+endif(COMMAND CMAKE_POLICY)
```

Comment:
```suggestion
cmake_policy(SET CMP0135 NEW)
```

Reply:
* There is no need to check if the command `cmake_policy` exists, it is available in CMake >= 3.16
* Style should be consistent, no need for `endif(ARGS)`

Reply:
Thank you.  Done.